crates/bevy_input/src/touch.rs

```diff
@@ -3,38 +3,43 @@ use bevy_ecs::system::ResMut;
 use bevy_math::Vec2;
 use bevy_utils::HashMap;
 
-/// Represents a touch event
+/// A touch input event.
 ///
-/// Every time the user touches the screen, a new `Start` event with an unique
-/// identifier for the finger is generated. When the finger is lifted, an `End`
+/// This event is the translated version of the `WindowEvent::Touch` from the `winit` crate.
```

Comment:
I feel like this should be at the end of the doc comment, it's more of a side note, but the average user probably doesn't care or need to know that we wrap winit. It's still important to have this information obviously.